### PR TITLE
Deploying archimedes on a local mainnet fork 

### PR DESCRIPTION
**Pull request by yotamdaniel**
This PR contains 
- localDeploy.js script to create an instance of archimedes on a local fork + fund accounts with needed funds to create position
- There is an issue where we cant get pool balances when running on a local fork of mainnet. I've added a very hacky way to go around that... if anyone has a better idea I am all ears :P

### CODE REVIEW

File: contracts/Exchanger.sol [hunk 1]
```diff
@@ -196,7 +196,7 @@ contract Exchanger is AccessController, ReentrancyGuardUpgradeable, IExchanger,
         // get_dy(indexCoinSend, indexCoinRec, amount)
         _expected3CRV = _poolLvUSD3CRV.get_dy(0, 1, amountLvUSD);
 
-        /// Make sure expected3CRV is not too high!
+        // /// Make sure expected3CRV is not too high!
```
Comment (yotamdaniel) [inline contracts/Exchanger.sol:199]: remove extra comments

File: contracts/LeverageEngine.sol [hunk 2]
```diff
@@ -69,19 +69,19 @@ contract LeverageEngine is AccessController, ReentrancyGuardUpgradeable, UUPSUpg
     ) external nonReentrant returns (uint256) {
         uint256 lvUSDAmount = _parameterStore.getAllowedLeverageForPosition(ousdPrinciple, cycles);
         uint256 lvUSDAmountAllocatedFromArch = _parameterStore.calculateLeverageAllowedForArch(archAmount);
+        lvUSDAmountAllocatedFromArch = lvUSDAmountAllocatedFromArch + 10000000; /// add some safety margin
```
Comment (yotamdaniel) [inline contracts/LeverageEngine.sol:72]: Modify archAmount getter to include a slight buffer

File: scripts/localDeploy.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    console.log("------ owner address is", await context.owner.address);
+    console.log("------ curve pool address is", await context.curveLvUSDPool.address);
+    console.log("---- test calculateArchNeededForLeverage - ",
+        await context.parameterStore.calculateArchNeededForLeverage(ethers.utils.parseUnits("3.67")));
+}
+
+const deployScript = async () => {
+    // hacky way to go around pool balances not working on local instance.. skipPoolBalances = true
+    context = await buildContractTestContext(true);
+    // await context.parameterStore.changeArchToLevRatio(ethers.utils.parseUnits("1.0"));
```
Comment (yotamdaniel) [inline scripts/localDeploy.ts:50]: remove

File: test/CurveHelper.ts [hunk 4]
```diff
@@ -42,6 +42,7 @@ async function createMetapool (token, owner) {
     // We deployed a 3CRV/lvUSD pool - so we ask Curve Factory to look for pools that can deal with USDT/lvUSD
     // In the future this will be a fixed index we can query instead
     const poolAddress = await factoryCurveMetapool.find_pool_for_coins(address3CRV, token.address);
+    console.log("findPoolForCoins - %s", poolAddress);
```
Comment (yotamdaniel) [inline test/CurveHelper.ts:45]: remove

**Review by jasonstockman — APPROVED** [review]

**Pull request merged**